File: Soal-2/PRAK502-2410817310014-INDRA_SURYADILAGA.c
#include <stdio.h>
#include <math.h>
int hitung(int nilai1, int nilai2) {
    return nilai1 - nilai2;
}

int mutlak(int angka) {
    return fabs(angka);
}

int main() {
    int x1, y1, x2, y2;
    scanf("%d", &x1);
    scanf("%d", &y1);
    scanf("%d", &x2);
    scanf("%d", &y2);

    int Hasil = hitung(x1, x2) + hitung(y1, y2);
    printf("%d", mutlak(Hasil));

    return 0;
}

File: Soal-2/PRAK502-2410817310014-INDRA_SURYADILAGA.py
def hitung(nilai1, nilai2):
  return nilai1 - nilai2

def mutlak(angka):
  return abs(angka)

x1, y1, x2, y2 = map(int, input().split())
Hasil = hitung(x1, x2) + hitung(y1, y2)
print(mutlak(Hasil))